docs/advanced/settings.md

```diff
@@ -60,6 +60,20 @@ The following example will add disks matching the glob pattern `/dev/sd*` or `/d
 /dev/sd*,/dev/hd*
 ```
 
+## `auto-rotate-rke2-certs`
+
+This setting allows you to rotate RKE2 certificates automatically. It's disable by default.
+The field `expiringInHours` allows you to set when to rotate RKE2 certificates before they expire.
+The valid range is from `1` to `8759` hours.
+
```

Comment:
Can we also include the RKE2 link for more info? Thanks.

Reply:
I think we have to use Rancher link since we rely on Rancher setting. WDYT?

https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/manage-clusters/rotate-certificates

Reply:
Sounds great, but I suggest we put both, RKE2 page contains info of when the certs expire: https://docs.rke2.io/advanced#certificate-rotation

Reply:
Update it. Thanks.